src/app/modules/item/item-routing.module.ts

```diff
@@ -13,6 +14,12 @@ import { ItemEditComponent } from './pages/item-edit/item-edit.component';
         {
           path: 'details',
           component: ItemDetailsComponent,
+          children: [
+            {
+              path: '',
+              component: ItemContentComponent,
+            }
+          ]
```

Comment:
As I changed the router, probably you have handle to conflict properly. But actually you just have to do the same with the new "detail" page which is one level deeper.